### DISCUSSION TITLE
实验二中有头文件的样例无法识别头文件

### DISCUSSION THREAD

**Discussion by hodanna7** [arcsysu/SYsU-lang#33] [Q&A] [answered]
使用评测脚本给出的测试指令显示以下错误：
tester/function_test2020/12_getint.sysu.c:1:10: fatal error: 'sysy/sylib.h' file not found
#include <sysy/sylib.h>
         ^~~~~~~~~~~~~~
1 error generated.
应该如何解决？

**Comment by Yorkking** [comment, marked as answer]
请看[这条](https://github.com/arcsysu/SYsU-lang/discussions/23)。
